### ISSUE TITLE
rpower commands run against OpenPower BMC should be blocked if not supported

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#2948]
This issue is opened just to track `rpower` but I think we need testing buckets to expand coverage to include more of the xCAT commands (FYI @hu-weihua)  If a command is not supported, we should not allow it to go through...   

Firstly, the rpower man page and rpower help does not match.  The man page contains BMC and OpenBMC specific.  `rpower` without options does not display BMC related options, only openbmc.  

```
     OpenBMC specific:
       rpower noderange [on|off|reset|boot|stat|state|status]
```

```
[root@fs1 ~]# rpower frame45cn07 stat
frame45cn07: on
[root@fs1 ~]# rpower frame45cn07 suspend
frame45cn07: Error: Request data length invalid   <--- this is an ERROR, we should exit here..
frame45cn07: suspend                              <--- with the above error, we should not even print this line....  
[root@fs1 ~]# rpower frame45cn07 stat
frame45cn07: on
```

@hu-weihua Can we expand the testing coverage to exercise this and other sub commands?

**Comment by zet809** [comment]
hi, @xuweibj , will you pls take a look at this issue? Thx!

**Comment by xuweibj** [comment]
The issue was fixed in pull request #2951

**Comment by whowutwut** [comment]
Looks good...
```
xCAT STATUS: Version 2.13.4 (git commit 9000ae5d1eedc1c412d2a6f35bc0346ef33ad8af, built Mon May 15 09:37:50 EDT 2017)
```
Testing against OpenPower BMC nodes
``` 
[root@stratton01 ~]# rpower frame5u39 state
frame5u39: on
[root@stratton01 ~]# rpower frame5u39 suspend
frame5u39: Error: unsupported command rpower suspend for OpenPower
[root@stratton01 ~]# rpower frame5u39 wake
frame5u39: Error: unsupported command rpower wake for OpenPower
```

Testing against OpenPower openbmc nodes:
```
[root@stratton01 ~]# rpower p9euh01 state
This openbmc related function is unsupported and disabled. To bypass, run the following:
        export XCAT_OPENBMC_DEVEL=YES
[root@stratton01 ~]# export XCAT_OPENBMC_DEVEL=YES
[root@stratton01 ~]# rpower p9euh01 state
Warning: Currently running development code, use at your own risk.  Unset XCAT_OPENBMC_DEVEL to disable.
p9euh01: on
[root@stratton01 ~]# rpower p9euh01 suspend
Warning: Currently running development code, use at your own risk.  Unset XCAT_OPENBMC_DEVEL to disable.
Unsupported command: rpower suspend
[root@stratton01 ~]# rpower p9euh01 wake
Warning: Currently running development code, use at your own risk.  Unset XCAT_OPENBMC_DEVEL to disable.
Unsupported command: rpower wake
[root@stratton01 ~]# rpower p9euh01 softoff
Warning: Currently running development code, use at your own risk.  Unset XCAT_OPENBMC_DEVEL to disable.
Unsupported command: rpower softoff
```
